core/arch/arm/include/kernel/generic_boot.h

```diff
@@ -42,5 +42,7 @@ struct ns_entry_context *generic_boot_core_hpen(void);
 #endif
 
 void *get_dt_blob(void);
+void *get_sec_dt_blob(void);
+void *get_ns_dt_blob(void);
```

Comment:
How about s/dt_blob/dtb/ everywhere for conciseness (and because we already use `CFG_*_DTB`)

Reply:
Please add comments to describe the functions.

Reply:
I asked for s/dt_blob/dtb/, not for s/sec/secure/. Do you like long names so much? ;-)
The shorter the better as long as it's still explicit IMO. `get_sec_dtb()` is as good as it gets I would say.

```C
/*
 * Returns embedded DTB if present, otherwise external DTB if it is in secure
 * memory, otherwise NULL.
 */
void *get_sec_dtb(void);

/* Returns external DTB if present and in non-secure memory, otherwise NULL */
void *get_ns_dtb(void);

/* Returns secure DTB if present, then non-secure DTB, then NULL */
void *get_dtb();
```

It is better to document the functions in the header files IMO.

Reply:
ok :)